ctapipe/containers.py

```diff
@@ -118,8 +123,35 @@ class HillasParametersContainer(Container):
     width_uncertainty = Field(nan * u.m, "uncertainty of width", unit=u.m)
     psi = Field(nan * u.deg, "rotation angle of ellipse", unit=u.deg)
 
-    skewness = Field(nan, "measure of the asymmetry")
-    kurtosis = Field(nan, "measure of the tailedness")
+
+class HillasParametersContainer(BaseHillasParametersContainer):
+    container_prefix = "hillas"
+    lon = Field(
+        nan * u.deg,
+        "centroid longitude in a sky frame e.g. the telescope frame",
+        unit=u.deg,
```

Comment:
if we stick with this renaming, it might be good to change the help string to say "longitude angle in a spherical system centered on the pointing position (TelescopeFrame)".   Also shouldn't it be fov_lon to be consistent with the Frame? (I don't really mind either way)

Reply:
+1 for `fov_lon`